api/ccapi/include/layer.h

```diff
@@ -26,26 +26,6 @@
 namespace ml {
 namespace train {
 
-/**
- * @brief     Enumeration of layer type
- */
-enum class LayerType {
```

Comment:
lets not delete this as we intend to support both enum and string.

Reply:
Done